pages/_app.js

```diff
@@ -1,14 +1,3 @@
-import GlobalStyle from "@/styles";
-import Head from "next/head";
-
-export default function App({ Component, pageProps }) {
-  return (
-    <>
-      <GlobalStyle />
-      <Head>
-        <title>Capstone Project</title>
-      </Head>
-      <Component {...pageProps} />
-    </>
-  );
+export default function App({ Component, pageProps: { pageProps } }) {
+  return <Component {...pageProps} />;
 }
```

Comment:
Why did you delete the `Head`?

```suggestion
export default function App({ Component, pageProps }) {
  return (
    <>
      <Head>
        <title>Capstone Project</title>
      </Head>
      <Component {...pageProps} />
    </>
  );
}
```

Reply:
actually i dont know im gonna fix that